website/index.html

```diff
@@ -254,21 +254,35 @@
 
     <div class="row">
       <div class="section-header col-12">
-        <div class="header-text">About Us</div>
+          <a name="message">
+        <div class="header-text">A Message to Our Participants</div>
+          </a>
         <div class="header-bar"></div>
       </div>
 
       <div class="col-12 about-text">
+
         <p>
```

Comment:
On my phone, but is this tag also supposed to be commented out?

Reply:
which tag? the about-text tag should be there if that's what you mean